.github/workflows/ci.yml

```diff
@@ -269,18 +269,104 @@ jobs:
       image: falcosecurity/driverkit:v0.10.1
     if: needs.paths-filter.outputs.driver_changed == 'true'
     steps:
-      - name: Test driver build on linux 2.6.32 
+      - name: Test driver build on linux 2.6.32
         run: |
           driverkit docker --kernelrelease 2.6.32-754.el6.x86_64 --target centos --output-module /tmp/libs.ko --driverversion $GITHUB_SHA --loglevel debug --kernelurls https://download.falco.org/fixtures/libs/kernel-devel-2.6.32-754.el6.x86_64.rpm
-          
+
       - name: Test driver build on linux 3.x
         run: |
           driverkit docker --kernelrelease 3.10.0-957.el7.x86_64 --target centos --output-module /tmp/libs.ko --driverversion $GITHUB_SHA --loglevel debug --kernelurls https://download.falco.org/fixtures/libs/kernel-devel-3.10.0-957.el7.x86_64.rpm
-    
+
       - name: Test driver build on linux 4.x
         run: |
           driverkit docker --kernelrelease 4.18.0-305.25.1.el8_4.x86_64 --target centos --output-module /tmp/libs.ko --output-probe=/tmp/libs.o --driverversion $GITHUB_SHA --loglevel debug --kernelurls https://download.falco.org/fixtures/libs/kernel-devel-4.18.0-305.25.1.el8_4.x86_64.rpm
-     
+
       - name: Test driver build on linux 5.x
         run: |
           driverkit docker --kernelrelease 5.19.12-arch1-1 --target arch --output-module /tmp/libs.ko --output-probe=/tmp/libs.o --driverversion $GITHUB_SHA --loglevel debug  --kernelurls https://download.falco.org/fixtures/libs/linux-headers-5.19.12.arch1-1-x86_64.pkg.tar.zst
+
+  run-e2e-tests-amd64:
```

Comment:
Are we able to enqueue this job to the other [system_deps, bundled_deps, system_deps_w_chisels] matrix? So that we save building everything again, that is time costly.

Reply:
I think it is just a matter of adding the right volumes and adding a step calling `make e2e-tests`

Reply:
That was my original approach, but then @LucaGuerra asked to separate them because the main ones are required and the `e2e-tests` are not thoroughly tested yet 🤷🏻‍♂️

Reply:
Uh i see the value; i am not sure though, because we already have too many builds in parallel and the CI is getting bigger and slower.
@LucaGuerra @leogr @Andreagit97 wdyt?

Reply:
This is a good point on one side I'm not sure about the stability since they are new, so I wouldn't put them as required from the first day, on the other side we have problems with the CI time :thinking: Maybe we can try them, I don't know 2 weeks, and then merge them with the others, WDYT?

Reply:
I would love to have them today as they are here so all other PRs can benefit from them! In the future we can move them everywhere we want :D